<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientForm</class>
 <widget class="QMainWindow" name="ClientForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClientForm</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Your message:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="messageLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="sendMessageButton">
        <property name="text">
         <string>Send to all</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="privateMessageButton">
        <property name="text">
         <string>Send private</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QTextBrowser" name="commonChatBrowser"/>
      <widget class="QTextBrowser" name="privateChatBrowser"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>607</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain_Menu">
    <property name="title">
     <string>Main Menu</string>
    </property>
    <addaction name="actionOpen_another_client"/>
    <addaction name="actionCloseThisClient"/>
   </widget>
   <addaction name="menuMain_Menu"/>
  </widget>
  <action name="actionOpen_another_client">
   <property name="text">
    <string>Open another client</string>
   </property>
  </action>
  <action name="actionCloseThisClient">
   <property name="text">
    <string>Close this client</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
